gateway/utils.py

```diff
@@ -37,10 +37,15 @@ def get_swagger_url_by_logic_module(module: LogicModule) -> str:
     :param LogicModule module: the logic module (service)
     :return: OpenAPI schema URL for the logic module
     """
-    return '{}/{}/{}.{}'.format(
-        module.endpoint, SWAGGER_LOOKUP_PATH,
-        SWAGGER_LOOKUP_FIELD, SWAGGER_LOOKUP_FORMAT
-    )
+    if module.docs_endpoint:
+        return '{}/{}/{}.{}'.format(
+            module.endpoint, module.docs_endpoint,
+            SWAGGER_LOOKUP_FIELD, SWAGGER_LOOKUP_FORMAT)
+    else:
+        return '{}/{}/{}.{}'.format(
+            module.endpoint, SWAGGER_LOOKUP_PATH,
+            SWAGGER_LOOKUP_FIELD, SWAGGER_LOOKUP_FORMAT
+        )
```

Comment:
You could use a variable to hold the lookup path value instead

Reply:
Maybe it should be the whole path to the file instead of part of it. What do you think?